lib/panmaster/include/panmaster/panmaster.h

```diff
@@ -37,7 +38,7 @@ struct panmaster_node {
     };
     uint8_t  index;          /*!< Index into vector */
     uint16_t slot_id;        /*!< slot_id, stored in flash if permanent */
-    struct pan_image_version fw_ver;
+    pan_image_version fw_ver;
```

Comment:
hehe, undo this and use struct pan_image_version

Reply:
Done